v2/index.html

```diff
@@ -1618,19 +1618,17 @@ <h4>Grace Notes</h4>
             </p>
             <p>
                 For a single acciaccatura the lower-case character <code>g</code>
-                MUST be used. This character MUST be specified in the order given above;
-                that is, it MUST precede the note name, and MUST also precede all other
-                optional attributes of the note. Consecutive single acciaccatura
-                SHOULD NOT occur.
+                MUST be used. This character MUST precede the note name, and MUST
+                also precede all other attributes of the note. Consecutive single
+                acciaccatura MUST NOT occur.
             </p>
             <p>
                 For a single appogiatura note, the lower-case character <code>q</code>
-                MUST be used. This character MUST be specified in the order given above;
-                that is, it MUST precede the note name, and MUST also precede all other
-                optional attributes of the note.
+                MUST be used. This character MUST precede the note name, and MUST
+                also precede all other optional attributes of the note.
             </p>
             <p>
-                Multple appogiatura notes SHOULD be encoded using an <a href="#appogiatura-groups">
+                Consecutive appogiatura notes SHOULD be encoded using an <a href="#appogiatura-groups">
```

Comment:
I am not sure what the Slack discussion was, but if we want to disallow them it should be changed to
```suggestion
                Consecutive appogiatura notes MUST be encoded using an <a href="#appogiatura-groups">
```